doc/api/path.markdown

```diff
@@ -62,12 +63,14 @@ path.dirname('/foo/bar/baz/asdf/quux')
 '/foo/bar/baz/asdf'
 ```
 
-## path.extname(p)
+## path.extname(path)
 
 Return the extension of the path, from the last '.' to end of string
 in the last portion of the path.  If there is no '.' in the last portion
 of the path or the first character of it is '.', then it returns
-an empty string.  Examples:
+an empty string. `path` must be a string.
```

Comment:
The docs should definitely be consistent but that can be done in a separate PR, I think. The entire path.markdown file is inconsistent with the other docs it appears.